src/Unity.Mathematics/Geometry/MinMaxAABB.cs

```diff
@@ -1,49 +1,332 @@
 using System;
+using System.Runtime.CompilerServices;
 using static Unity.Mathematics.math;
 
 namespace Unity.Mathematics.Geometry
 {
+    /// <summary>
+    /// Axis aligned bounding box (AABB) stored in min/max form.
+    /// </summary>
+    /// <remarks>
+    /// Axis aligned bounding boxes (AABB) are boxes where each side is parallel with one of the Cartesian coordinate axes
+    /// X, Y, and Z. AABBs are useful for approximating the region an object (or collection of objects) occupy and quickly
```

Comment:
"AABBs are useful for approximating the region an object (or collection of objects) occupy" -> occupies